chirp/drivers/tdh8.py

```diff
@@ -1519,12 +1519,20 @@ def _get_settings(self):
         basic.append(rs)
 
         if self.MODEL != "RT-730":
-            rs = RadioSetting("rogerprompt", "Roger",
-                              RadioSettingValueBoolean(_settings.rogerprompt))
+            if self.MODEL in H3_LIST:
+                # H3 uses roger-beep list
+                rs = RadioSetting("rogerprompt", "Roger",
+                                  RadioSettingValueList(
+                                      self._roger_list,
+                                      current_index=_settings.rogerprompt))
+            else:
```

Comment:
This will allow H8, not H3 because they took the path above. RT-370 will not make it here because of the first if.

If you remove the top if and change this `else` to `elif self.MODEL != 'RT-370'` then I think it's the same without the extra nesting. Further, you could use something like `elif self.MODEL.startswith('TD-H8')` if you want it a little more future proof or something.

Anyway, not a big deal, I just don't really like nested `if`s when they look like this